scripts/readme/readme.py

```diff
@@ -275,7 +275,8 @@ def write_readme(workflow_telemetry, readme_telemetry):
     input_glob_readme = [
         "examples/flows/**/README.md",
         "examples/connections/**/README.md",
-        "examples/tutorials/**/chat*.md",
+        "examples/tutorials/e2e-development/*.md",
```

Comment:
Why do you need to change this?

Reply:
It is because that we aim to show the index in examples/readme.md and tried to run the script examples/readme.py, but encountered some issues. Consequently, we reached out to @crazygao  for help.

Reply:
Not all tutorials are included in readme generation. I made changes to take this README into account

Reply:
then why do you delete the chat*.md?

Reply:
what's the update of this?

Reply:
they are the same, chat-with-pdf is also contained inside e2e-development folder.

why this is strict, because we actually don't want user to add new readme under examples/tutorials folder, this folder is originally meant for notebooks.